site/package.json

```diff
@@ -136,8 +134,7 @@
     "typescript": "4.8.2",
     "webpack": "5.74.0",
```

Comment:
Should we use Vite for production too?

Reply:
Yes, but I have to run some experiments to check the build and build size.